apps/omg_conformance/test/support/conformance/merkle_proofs.ex

```diff
@@ -58,7 +58,11 @@ defmodule Support.Conformance.MerkleProofs do
   defp call_contract(contract, signature, args, return_types) do
     data = ABI.encode(signature, args)
 
-    {:ok, return} = Ethereumex.HttpClient.eth_call(%{to: Encoding.to_hex(contract), data: Encoding.to_hex(data)})
+    from = Application.fetch_env!(:omg_eth, :eth_call_from_address)
+
+    {:ok, return} =
+      Ethereumex.HttpClient.eth_call(%{from: from, to: Encoding.to_hex(contract), data: Encoding.to_hex(data)})
+
```

Comment:
this seems a bit unstructured, you're accessing the configuration from too deep. perhaps all that's needed to make this call should be sent as a function argument

Reply:
This argument is irrelevant, we could easily hardcode anything into it. I would rather leave it here than pretend it's something meaningful that has to be passed from higher levels.